### PR TITLE
fix some test cases failed

### PR DESCRIPTION
**Pull request by xuchunmei000**

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, test this please]

File: tests/python/test_clang_complex.c [hunk 1]
```diff
@@ -20,7 +20,7 @@ struct ConfigLeaf {
   u32 bpfdev_ip;
   u32 slave_ip;
 };
-BPF_TABLE("array", struct ConfigKey, struct ConfigLeaf, config_map, 1);
+BPF_TABLE("array", u32, struct ConfigLeaf, config_map, 1);
```
Comment (yonghong-song) [inline tests/python/test_clang_complex.c:23]: Could you change other usages of `struct ConfigKey` as well and eventually remove struct ConfigKey definition? For example, `struct ConfigKey cfg_key = {.index = 0};` can be changed to `u32 cfg_key = 0`.

Comment (xuchunmei000) [reply tests/python/test_clang_complex.c:20]: done

**Comment by yonghong-song** [comment]
[buildbot, test this please]

**Pull request merged**